climada/hazard/low_flow.py

```diff
@@ -462,23 +553,30 @@ def _init_centroids(data_x, centr_res_factor=1):
 
 def unique_clusters(data):
     """identify unqiue clustes based on clusters in 3 dimensions and set unique
-    cluster_id"""
+    cluster_id
+
+    Parameters:
+        data (pandas.DataFrame): contains monthly gridded data of days below threshold
```

Comment:
is data a good name here? data is very non-explicit. This comment probably applies to the whole file and to several methods.

Reply:
you are right, i rename all `data` that are an DataFrame object to `data_df` and all `data` that are an xarray object to `data_xarray`

Reply:
Hmmm not sure if this is really the optimal solution. What data is it? Data is a too generic name to mean anything. Is it houses? Hurricanes? apple juice flavors?

Reply:
I called it data, because they contain several things.
As an xarray, it contains the variable "dis" + coordinates; which originally is "dis=discharge" as imported from the netcdf file. over time, it is however changing, so i could call it rather `dis_xarray` and later `days_below_thresh`? 

as a DataFrame, it contains various columns including days below threshold, date, lan, lot, cluster_ID, etc., basically all variables required to define and populate the hazard set. That's why I just called it `data`. Any suggestions how to call it different from `data_df`?

Reply:
Well, the data still represents something uniform, right? In your case, you say it is the hazard, which here is the low water flow? Then the name of the data variable should imo. reflect that. For instance, data_haz, or flow_data, or, df_haz, ... Would you agree? On another note, I think it is suboptimal if the content of a variable changes in 'spirit' during the code (inside the same method/class) without it being clear from other variable properties.